lib/manager/gradle-wrapper/extract.js

```diff
@@ -0,0 +1,43 @@
+module.exports = {
+  extractPackageFile,
+};
+
+function extractPackageFile(fileContent) {
+  logger.debug('gradle-wrapper.extractPackageFile()');
+  const lines = fileContent.split('\n');
+
+  let lineNumber = 0;
+  for (const line of lines) {
+    const match = line.match(
+      /^distributionUrl=https\\:\/\/services.gradle.org\/distributions\/gradle-((\d|\.)+)-(bin|all)\.zip\s*$/
+    );
+    if (match) {
+      const dependency = {
+        purl: 'pkg:gradle',
+        depType: 'gradle-wrapper',
+        depName: 'gradle',
+        currentValue: match[1],
+        lineNumber,
+        versionScheme: 'semver',
+        /* gradle specifics ... */
+        type: match[3],
```

Comment:
Let's give this a more gradle-specific name to avoid potential namespace conflicts later, e.g. `gradleWrapperType`